src/validations/rules/ssp.sch

```diff
@@ -1645,8 +1645,12 @@
                 diagnostics="uses-correct-param-descriptions-diagnostic"
                 id="uses-correct-param-descriptions"
                 role="error"
-                test="$profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID][o:label eq $paramValue or o:constraint/o:description/o:p eq $paramValue]">A
-                FedRAMP SSP must use correct parameter descriptions.</sch:assert>
+                test="
+                    if ($profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID]/o:constraint)
+                    then
+                        ($profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID][o:constraint/o:description/o:p eq $paramValue])
+                    else
+                        (true())">A FedRAMP SSP must use correct parameter descriptions.</sch:assert>
```

Comment:
The issue which spawned your PR says the following but you have not accomplished that task.

> Create one or more Schematron assertions which compare all SSP `<implemented-requirement>` `<set-parameter>` `<value>` to the value declared in the corresponding (if present) `<control>` `<param>` `<constraint>` value declared in associated resolved catalog based on the SSP FIPS 199 categorization.